onset_maproom/calc.py

```diff
@@ -131,7 +209,34 @@ def cess_date(
     min_dry_days, 
     time_coord="T"
 ):
-
+    """ Computes single cessation date from daily soil moisture data.
+
+        Cessation Date finds the first day of the first dry spell where soil moisture falls 
+        below the defined dry threshold value for a minimum number of days.
+
+        Parameters
```

Comment:
Have Parameters and Returns consistent in their phrasing between onset_date and cess_date